Covid/MainScreen/MainViewController.swift

```diff
@@ -158,13 +185,14 @@ extension MainViewController {
                 }
             }
 
-            action()
             if Defaults.pushToken == nil {
                 observer = Defaults.observe(\.pushToken) { _ in
                     DispatchQueue.main.async {
                         action()
                     }
                 }
+            } else {
+                action()
```

Comment:
@anonymcek registerUser API call sa pri prvom starte volal 2x za sebou.
1. `action()` by default
2. pri setupnuti observera na push token

Reply:
takto som to mal a z nejakeho dovodu som to prehodil, ale netusim preco